frameworks/kafka/universe/config.json

```diff
@@ -698,13 +765,19 @@
           "title": "log.cleaner.io.max.bytes.per.second",
           "description": "The log cleaner will be throttled so that the sum of its read and write i/o will be less than this value on average",
           "type": "number",
-          "default": 1.7976931348623157e+308
+          "default": 1.80E+308
         },
         "log_cleaner_min_cleanable_ratio": {
           "title": "log.cleaner.min.cleanable.ratio",
           "description": "The minimum ratio of dirty log to total log for a log to eligible for cleaning",
-          "type": "number",
-          "default": 0.5
+          "type": "double",
+          "default": "0.5"
```

Comment:
Should this not just be `"number"` with a value of `0.5`? (not a string?)

Reply:
Whoops! Missed that one, thanks.

Reply:
Ah hah! That would explain my failed CI. My script to build all these didn't account for "double"

Reply:
FWIW it may make sense to also include the script in this PR. I imagine it'll be useful for resyncing as options are added/changed in Kafka over time.